Argo/Globals/fold.swift

```diff
@@ -0,0 +1,7 @@
+import Runes
+
+func foldM<S: SequenceType, U>(sequence: S, initial: U, combine: (U, S.Generator.Element) -> U?) -> U? {
```

Comment:
Not sure where you ever landed on using names from haskell, but this name makes no sense otherwise.